src/stores/session.ts

```diff
@@ -45,6 +47,8 @@ export const useSessionStore = defineStore('session', {
 	getters: {
 		full_name: (state): string => `${state.user?.first_name ?? ''} ${state.user?.last_name ?? ''}`,
 		getUser: (state): User => new User(state.user),
+		student_user_courses: (state) => state.user_courses.filter(c => c.role === 'student'),
+		instructor_user_courses: (state) => state.user_courses.filter(c => c.role === 'instructor'),
```

Comment:
I don't like this. I only see this being useful for the UserCourses page, and not really anywhere else... so just filter on that one page as we had before.